<ui version="4.0" >
 <class>cfgFilesystem</class>
 <widget class="QWidget" name="cfgFilesystem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Filesystem Configuration</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Filesystem Access for this Mobile Phone</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="Q3WidgetStack" name="fs_stack" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="voidpage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>578</width>
        <height>122</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="p2k" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" native="1" name="kActiveLabel1" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>11</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>11</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel2" >
             <property name="text" >
              <string>Vendor</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel3" >
             <property name="text" >
              <string>Product ID</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>11</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="KLineEdit" native="1" name="kcfg_p2k_vendor" />
           </item>
           <item>
            <widget class="KLineEdit" native="1" name="kcfg_p2k_prodid" />
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>121</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="obexftp" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>11</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>11</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel4" >
             <property name="text" >
              <string>Transport type</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel5" >
             <property name="text" >
              <string>Device name (use "&lt;i>nodevice&lt;/i>" for auto/null devices)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel6" >
             <property name="text" >
              <string>Channel/Port (0 for auto)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>11</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="KComboBox" name="kcfg_obex_transp" >
             <item>
              <property name="text" >
               <string>IrDA</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Inet</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Custom</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Bluetooth</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>USB</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="KLineEdit" native="1" name="kcfg_obex_device" />
           </item>
           <item>
            <widget class="KIntNumInput" native="1" name="kcfg_obex_port" />
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="KComboBox" name="kcfg_fstype" >
     <item>
      <property name="text" >
       <string>No Filesystem</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Motorola P2K (USB)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Obex</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
  </customwidget>
  <customwidget>
   <class>Q3WidgetStack</class>
   <extends>Q3WidgetStack</extends>
   <header>q3widgetstack.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QWidget</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >klineedit.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >kcombobox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
